### PR TITLE
add troubleshooting reports section

### PR DESCRIPTION
**Pull request by eddycharly**
## Related issue #

Fixes https://github.com/kyverno/website/issues/859

### CODE REVIEW

File: content/en/docs/Troubleshooting/_index.md [hunk 1]
```diff
@@ -153,3 +153,11 @@ After gathering this information, [create an issue](https://github.com/kyverno/k
 **Solution**: There can be many reasons why a policy may fail to work as intended, assuming other policies work. One of the most common reasons is that the API server is sending different contents than what you have accounted for in your policy. To see the full contents of the AdmissionReview request the Kubernetes API server sends to Kyverno, add the `dumpPayload` [container flag](/docs/installation/customization/#container-flags) set to `true` and check the logs. This has performance impact so it should be removed or set back to `false` when complete.
 
 The second most common reason policies may fail to operate per design is due to variables. To see the values Kyverno is substituting for variables, increase logging to level `4` by setting the container flag `-v=4`. You can `grep` for the string `variable` (or use tools such as [stern](https://github.com/stern/stern)) and only see the values being substituted for those variables.
+
+## Admission reports are stacking up
+
+**Symptom**: Admission reports keep accumulating in the cluster, taking more and more ETCD space and slowing down requests.
```
Comment (chipzoller) [inline content/en/docs/Troubleshooting/_index.md:159]: etcd

File: content/en/docs/Troubleshooting/_index.md [hunk 2]
```diff
@@ -153,3 +153,11 @@ After gathering this information, [create an issue](https://github.com/kyverno/k
...
+
+**Diagniose**: Please follow the [troubleshooting docs](https://github.com/kyverno/kyverno/blob/main/docs/dev/troubleshooting/reports.md) to determine if you are affected by this issue.
```
Comment (chipzoller) [inline content/en/docs/Troubleshooting/_index.md:161]: Diagnose

File: content/en/docs/Troubleshooting/_index.md [hunk 3]
```diff
@@ -153,3 +153,11 @@ After gathering this information, [create an issue](https://github.com/kyverno/k
...
+
+**Solution**: Admission reports can accumulate if the reports controller is not working properly so the first thing to check is if the reports controller is working as expected. If the controller works as expected, another potential cause is that it fails to aggregate admission reports fast enough, this usually happens when the controller is throttled, you can fix this by increasing `QPS` and `Burst` for the controller by setting ``--clientRateLimitQPS=300` and `--clientRateLimitBurst=300`.
```
Comment (chipzoller) [inline content/en/docs/Troubleshooting/_index.md:163]: I am a user reading this. How do I determine if my reports controller is "working properly" or not? You should also mention the CronJobs as these should take care of it. And those flags are already defaults now, right? So setting these values doesn't change anything from defaults.

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: content/en/docs/Troubleshooting/_index.md [hunk 4]
```diff
@@ -153,3 +153,11 @@ After gathering this information, [create an issue](https://github.com/kyverno/k
...
+
+## Admission reports are stacking up
+
+**Symptom**: Admission reports keep accumulating in the cluster, taking more and more etcd space and slowing down requests.
+
+**Diagnose**: Please follow the [troubleshooting docs](https://github.com/kyverno/kyverno/blob/main/docs/dev/troubleshooting/reports.md) to determine if you are affected by this issue.
+
+**Solution**: Admission reports can accumulate if the reports controller is not working properly so the first thing to check is if the reports controller is running and does not continuously restarts. If the controller works as expected, another potential cause is that it fails to aggregate admission reports fast enough, this usually happens when the controller is throttled, you can fix this by increasing `QPS` and `Burst` for the controller by setting ``--clientRateLimitQPS=500` and `--clientRateLimitBurst=500`.
```
Comment (chipzoller) [inline content/en/docs/Troubleshooting/_index.md:163]: "...and no restarts are observed."

You have a tendency to write [run-on sentences](https://www.csuchico.edu/slc/_assets/documents/writing-center-handouts/run-on-sentences-handout.pdf) in English. If two clauses are fully independent, they are separate sentences. For example (with fix):

"If the controller works as expected, another potential cause is that it fails to aggregate admission reports fast enough. This usually happens when the controller is throttled. You can fix this by increasing QPS and burst rates for the controller by setting `--clientRateLimitQPS=500` and `--clientRateLimitBurst=500`."

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: content/en/docs/Troubleshooting/_index.md (hunk 2 above)
Comment (eddycharly) [reply content/en/docs/Troubleshooting/_index.md]: fixed

File: content/en/docs/Troubleshooting/_index.md (hunk 4 above)
Comment (eddycharly) [reply content/en/docs/Troubleshooting/_index.md]: Thanks, fixed. I'm definitely not a good writer :(

Comment (chipzoller) [reply content/en/docs/Troubleshooting/_index.md]: Fix isn't complete. Please see my suggestion carefully.

Comment (eddycharly) [reply content/en/docs/Troubleshooting/_index.md]: 🤦

Comment (eddycharly) [reply content/en/docs/Troubleshooting/_index.md]: fixed 🤞

**Review by chipzoller — APPROVED** [review]

**Pull request merged**